### ISSUE TITLE
Better help

### ISSUE THREAD

**Issue by Firigion** [Ghoulboy78/Scarpet-edit#98]
The help menu is getting out of hand. It was  concived when the app was much smaller, but we no have 10 pages of very condensed information. I'd suggest splitting up the help menu into sections by subject (shapes, setting blocks, brushes, selection, ect).

Just leaving the issue here as a reminder for the future.

**Comment by Ghoulboy78** [comment]
But how would we go about splitting into sections? would we just manually assign command to a category? cos that feels like pain ngl...

**Comment by Firigion** [comment]
Well, we can do something along the lines of splitting by the start of the string or something like that. Almost all the help strings start with `'help_cmd_`, so taht's a start. Then there's all the `..._brush_...` ones, and there's not so many more. I'd be up for changing those, it's 36 in total, quite easy to clasify. We'd need to discuss the formatting, tho.

**Comment by Ghoulboy78** [comment]
yes, cos we also need to discuss how exactly to implement it as well, as well as a protocol to both categorize future commands as well as add new categories.
